boards/FSM/Src/Io/Io_FlowMeters.c

```diff
@@ -0,0 +1,84 @@
+#include "main.h"
+#include "Io_SharedFrequencyOnlyPwmInput.h"
+#include "Io_FlowMeters.h"
+#include <assert.h>
+
+#define PRIMARY_FLOW_METER_CHANNEL HAL_TIM_ACTIVE_CHANNEL_1
+#define SECONDARY_FLOW_METER_CHANNEL HAL_TIM_ACTIVE_CHANNEL_2
+
+struct FlowMeters_Context
+{
+    struct FreqOnlyPwmInput *pwm_input;
+    size_t                   elapsed_tick;
```

Comment:
A comment about elapsed tick would be nice

Reply:
moved elapsed tick to `Io_SharedFreqOnlyInput.c` instead